CHANGELOG.md

```diff
@@ -25,6 +25,8 @@ All notable changes to this project will be documented in this file.
   (<https://github.com/openvinotoolkit/training_extensions/pull/4053>)
 - Reduce tiling unittests resource consumption
   (<https://github.com/openvinotoolkit/training_extensions/pull/4073>)
+- Upgrade OpenVINO and NNCF dependencies
```

Comment:
I think it is not necessary to put upgrade of the versions in the changelog, what do you think?

Reply:
Updated